docs/developers/reference/api/eth-sendrawtransaction.mdx

```diff
@@ -46,8 +46,10 @@ The 32-byte transaction hash, or the zero hash if the transaction is not yet ava
 
 ## Example
 
+### Request
+
 <Tabs>
```

Comment:
I'm fine with diving up the Example section into Request and Response, but it doesn't seem to make sense to leave the tabs in. Currently we only have one tab in each `<Tabs>`?